### PR TITLE
draw mined item for laser and blast drills

### PR DESCRIPTION
**Pull request by Slava0135**
I know that Anuke some day removed this indicator from the said drills. But imo it's quite handy and players will be happy about having it.

If you don't like it for some reason I can do some changes to this.

Screenshot from 2020-12-09 20-23-52

### CODE REVIEW

**Comment by Slava0135** [comment]
I mean just look - thermal pump has the indicator while electric drill - don't

**Review by Quezler — APPROVED** [review]

**Comment by Slava0135** [comment]
Perhaps it should be little bit bigger and (or) be configurable - you could specify the rectangle's side length

**Comment by Anuken** [comment]
If you're going to make a PR for this, at least fix the indicators so they fit the drill size.

**Comment by Slava0135** [comment]
Screenshot from 2020-12-17 10-24-44

**Comment by Slava0135** [comment]
Screenshot from 2020-12-17 10-44-49

**Review by Quezler — APPROVED** [review]

File: core/src/mindustry/world/blocks/production/Drill.java [hunk 1]
```diff
@@ -304,7 +305,7 @@ public void draw(){
 
             if(dominantItem != null && drawMineItem){
                 Draw.color(dominantItem.color);
-                Draw.rect("drill-top", x, y);
+                Draw.rect(itemRegion, x, y);
```
Comment (Quezler) [inline core/src/mindustry/world/blocks/production/Drill.java:308]: you've missed one `"drill-top"` at line 84 ^

**Pull request closed without merging**